src/libs/actions/IOU.js

```diff
@@ -1023,13 +1023,6 @@ function getPayMoneyRequestParams(chatReport, iouReport, recipient, paymentMetho
                 },
             },
         },
-        {
```

Comment:
This was redundant in `successData`, because it's already included in `optimisticData` on line 987